installer/roles/image_build/templates/Dockerfile.j2

```diff
@@ -170,13 +170,11 @@ ADD {% if build_dev|bool %}installer/roles/image_build/files/{% endif %}rsyslog.
 
 ## File mappings
 {% if build_dev|bool %}
-ADD tools/docker-compose/launch_awx.sh /usr/bin/launch_awx.sh
```

Comment:
Doesn't the k8s installer rely on this to exist at `/usr/bin` ?

```
installer/roles/image_build/templates/Dockerfile.j2
175:ADD tools/docker-compose/launch_awx.sh /usr/bin/launch_awx.sh
185:ADD launch_awx.sh /usr/bin/launch_awx.sh
247:CMD /usr/bin/launch_awx.sh
```

Reply:
for the prod env, yes... this only touches the dev env from a quick read